rcast.py

```diff
@@ -1,108 +1,129 @@
 #!/usr/bin/env python
 
-import urllib, urllib2, os, sys, thread, base64, signal, socket, SimpleHTTPServer, SocketServer, time, json, logging
+import urllib
+import urllib2
+import os
+import sys
+import thread
+import signal
+import socket
+import SimpleHTTPServer
+import SocketServer
+import time
+import json
+
 
 def signal_handler(signal, frame):
-        print('You pressed Ctrl+C, stopping. Casting may continue for some time.')
+        print(
+            'You pressed Ctrl+C, stopping. Casting may continue for some time.'
```

Comment:
Personally I'd say this formatting looks a bit weird, but if this is the way to go, then sure.